### PR TITLE
add convertTime function

### PR DESCRIPTION
**Pull request by bryanakitchen**
This takes total number of minutes and converts into hours and minutes when relevant.

### CODE REVIEW

**Review by ShaneUP1 — APPROVED** [review]
Looks good, have you had a look at how to implement the setting on this time part?

File: utils/convert-time.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    if (number < 60) {
+        return `${roundMinutes} minutes`
+    } else if (number === 60) {
+        return `${hours} hour`;
+    } else if (number > 60) {
+        return `${roundHours} hours and ${roundMinutes} minutes`
+    }
+
+  }
+
```
Comment (franco-ortega) [inline utils/convert-time.js:16]: I like how this work. The only part that I found a bit confusing and potentially repetitive was line 2:

const number = num;

Could num be used everywhere that number is used?

**Review by franco-ortega — APPROVED** [review]

File: utils/convert-time.js (hunk 1 above)
Comment (bryanakitchen) [reply utils/convert-time.js:16]: Updated.

**Review by bryanakitchen — COMMENTED** [review]
Updated

**Pull request merged**